spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
+                        "zmanimInformation":
+                        {
+                            "name": "",
+                            "url": ""
+                        },
+                        "information":
+                        {
+                            "name": "US Naval Observatory's Almanac for Computer algorithm",
+                            "url": "http://aa.usno.navy.mil/publications/docs/asa.php"
+                        }
+                    },
+                    "date": "2010-04-02",
+                    "location":
+                    {
+                        "name": "Lakewood, NJ",
+                        "lat": 40.09596,
+                        "long": -74.22213,
+                        "elevation": 0,
+                        "timeZone":
+                        {
+                            "offset": -5,
+                            "zoneInfoName": "America/New_York"
+                        }
+                    }
+                },
+                "tests":
+                [
+                    {
+                        "method": "sunset",
+                        "description": "Gets the exact sunset time",
+                        "extra":
```

Comment:
Can you clarify what is the intended function of this area?

Reply:
This is for extra information about the test case so that if we have a UI, we can show that. Nothing functional.

Reply:
I have similar reservations about this object.